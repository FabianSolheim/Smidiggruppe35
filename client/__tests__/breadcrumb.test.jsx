import React from "react";
import ReactDOM from "react-dom";
import BreadcrumbComponent from "../src/components/Breadcrumbs/breadcrumbs";
import { MemoryRouter } from "react-router-dom";

describe("breadcrumb test", () => {
  it("shows breadcrumbs", () => {
    const domElement = document.createElement("Breadcrumbs");
    ReactDOM.render(
      <MemoryRouter>
        <BreadcrumbComponent />
      </MemoryRouter>,
      domElement
    );
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
