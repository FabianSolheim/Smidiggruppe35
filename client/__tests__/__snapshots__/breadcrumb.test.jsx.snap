// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breadcrumb test shows breadcrumbs 1`] = `"<nav class=\\"MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-63vwl1-MuiTypography-root-MuiBreadcrumbs-root\\"><ol class=\\"MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol\\"></ol></nav>"`;
